code/datums/gamemode/gamemode.dm

```diff
@@ -250,6 +251,10 @@
 		F.process()
 	for(var/datum/role/R in orphaned_roles)
 		R.process()
+	if (runescape_pvp)
+		for (var/entry in runescape_fighters)
+			var/datum/runescape_fighter_data/the_data = runescape_fighters[entry]
```

Comment:
make this a proper role datum, there is absolutely no point or benefit in having two separate system, role datums are supposed to have a universal character

Reply:
The point is to have it be something much cheaper and independent from role datums (don't want to clog the round end scoreboard either)

Reply:
At the very least I moved all the stuff in mob.dm to its own files under modules.

Reply:
More importantly, we don't want it to interfere with other roles in any way. Some features interfere with players that already have a role and I don't want being flagged as a runescape fighter causing such interference.

Reply:
Then those features are the problem. 
What are they even? I am sorry, but no, I am not letting this new type of role not use the role datums framework.

Reply:
That's not a type of role, it's an bus-only toggle button that makes skulls appear on top of players who've attacked someone else. And also pacifies everyone standing outside of maint, this is all for the purpose of a bus that ancient has in mind. This won't become part of every rounds.

Reply:
the french fighting each other is good

Reply:
>it's as if bombermen players in bomberman arenas needed to have their own role datum
damn, that would be pretty cool. a separate area for roles that aren't really relevant if you're just wondering "was it valid for that guy to kill me," but you could still see stats like which "mini-role" killed who, which bomberman won the bomberround, etc